IfcCommunicationsAppliance Rail Import BIMQ 2021-07-21T07:21:13.7228368Z (#177)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/n/NumberOfCommonInterfaces_1DWcqAjZr43OGGCJpgy1j6/Documentation.md

> Indicates the types of common interfaces and their number in the device.

Reviewer: ```suggestion
Indicates the number of common interfaces on the device.
```
